File: JavaScript/armstrong-numbers.js
const isArmstrongNumber = (n) => {
    let numbers = String(n).split("").map(num => Number(num));
    
    let powers = numbers.map(num => Math.pow(num, numbers.length));
    let sum = powers.reduce((a, b) => a + b, 0);
    
    return n === sum
};

numbers = [
    9,
    10,
    153,
    154,
];

numbers.forEach(number => {
    console.log(`isArmstrongNumber(${number}) -> ${isArmstrongNumber(number)}`);
});

File: C#/Armstrong Numbers/ArmstrongNumbers.cs
using System;
using System.Linq;

public static class ArmstrongNumbers
{
    public static bool IsArmstrongNumber(int number)
    {
        var numbers = number.ToString().ToCharArray().Select(x => Convert.ToInt32(x.ToString()));
        var result = numbers.Aggregate(0, (acc, x) => acc + (int)Math.Pow(x, numbers.Count()));

        return number == result;
    }
}